api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -0,0 +1,95 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.List;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "institution")
+public class DbInstitution {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_id")
+  private long institutionId; // primary opaque key for DB use only
+
+  @Column(name = "api_id", nullable = false, unique = true)
```

Comment:
Which API? There are a number of them running around.

Reply:
the workbench's swagger api - do you have a different name preference?